build/README

```diff
@@ -27,15 +27,21 @@ Prerequisites to build tgz, debian and rpm packages:
 --------------------------------------------------------------------------------------------------
 
 Prerequisites to build autoexe DoliWamp package:
+
 > apt-get install wine q4wine
+
 > Launch "wine cmd" to check a drive Z: pointing to / exists.
+
 > Install InnoSetup
-   For example by running isetup-5.5.8.exe (https://www.jrsoftware.org)  https://files.jrsoftware.org/is/5/
+   For example by running innosetup-6.1.2.exe  (https://jrsoftware.org/isinfo.php)   https://files.jrsoftware.org/is/5/
```

Comment:
Did you made test with innosetup 6 to validate v6 is ok to build the .exe with the makepack-dolibarr.pl tool ?

Reply:
Not yet. Last check was with InnoSetup 6.0.4 and Dolibarr 12.0.3.
Let's leave it open until i made the check.